treeherder/etl/perf.py

```diff
@@ -214,20 +216,21 @@ def _load_perf_datum(job: Job, perf_datum: dict):
                 },
             )
 
-            (suite_datum, datum_created) = PerformanceDatum.objects.get_or_create(
-                repository=job.repository,
-                job=job,
-                push=job.push,
-                signature=signature,
-                push_timestamp=deduced_timestamp,
-                defaults={'value': suite['value'], 'application_version': application_version},
-            )
-            if suite_datum.should_mark_as_multi_commit(is_multi_commit, datum_created):
-                # keep a register with all multi commit perf data
-                MultiCommitDatum.objects.create(perf_datum=suite_datum)
+            if suite.get("value", None) is not None:
```

Comment:
From this I understand that for benchmark tests without a suite-level value there won't have performance datums?
Are the subtests affected in the same way?

Reply:
That's correct, for benchmarks without a suite-level value, there won't be a suite-level PerformanceDatum created for it. However, the subtests will still have a PerformanceDatum created for them since that happens on line 235, outside of this `if` block.